src/main/kotlin/racing/model/Racing.kt

```diff
@@ -1,5 +1,23 @@
 package racing.model
 
-class Racing(carCount: Int) {
-    var cars: List<Car> = List(carCount) { Car() }
+class Racing(carNames: List<String>, val tryGameCount: Int) {
+    var cars: MutableList<Car> = mutableListOf()
```

Comment:
아래와 같이 주생성자를 통해서,  Racing 객체의 프로퍼티를 초기화할 수 있도록하고 외부에서 `carNames`를 `List<Car>`로 가공해서 주입해주면 어떨까요?

```suggestion
class Racing(
    cars: List<Car> , 
    val tryGameCount: Int,
) {
```